test/e2e/main-e2e.test.js

```diff
@@ -27,6 +27,9 @@ before(async function () {
   this.timeout(process.env.TRAVIS || process.env.APPVEYOR ? 10 * 60 * 1000 : 30 * 1000);
   this.app = new Application({
     path: appPath,
+    env: {
+      FORCE_NO_WRONG_FOLDER: true,
```

Comment:
Don't give the 'Not in Application Folder' error for E2E tests....

Don't know how this worked before.